ansible/roles/vast/tasks/static.yaml

```diff
@@ -0,0 +1,82 @@
+- name: Info message
+  ansible.builtin.debug:
+    msg: "Using static binary install"
+- name: Stop vast service
+  ansible.builtin.service:
+    name: vast
+    state: stopped
+  ignore_errors: yes
+- name: Remove VAST debian package
+  ansible.builtin.apt:
+    name: "vast"
+    state: absent
+- name: Create vast group
+  ansible.builtin.group:
+    name: vast
+    state: present
+- name: Create vast user
+  ansible.builtin.user:
+    name: vast
+    groups: vast
+    shell: /sbin/nologin
+    append: yes
+    state: present
+    create_home: no
+
```

Comment:
Question/Nit: Is there a general guideline for when you use an empty line between top-level list entries in this YAML? Up to this point there are none, and from here on it's one per step.

Reply:
`ansible-lint` doesn't seem to care either way. The samples in official docs separate multi-line YAML array elements with an empty line.

Incidentally `ansible-lint` found a few other issues that I'm going to fix now. Wdyt of adding `ansible-lint` to the style check?

Reply:
Good idea.

Reply:
done